docs/reference/ml/anomaly-detection/apis/open-job.asciidoc

```diff
@@ -47,6 +47,17 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=job-id-anomaly-detection]
   (Optional, time) Controls the time to wait until a job has opened. The default
   value is 30 minutes.
 
+[[ml-open-job-response-body]]
+==== {api-response-body-title}
+
+`node`::
+  (string) The ID of the node that the job was opened on, or an empty string
```

Comment:
I recommend splitting this into two shorter sentences and putting the conditional info at the start of the second sentence:
```suggestion
  (string) The ID of the node that the job was opened on. If the job is allowed to 
```